src/main/java/com/javadiscord/javabot/commands/reaction_roles/HelpRole.java

```diff
@@ -0,0 +1,62 @@
+package com.javadiscord.javabot.commands.reaction_roles;
+
+import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
+import net.dv8tion.jda.api.events.message.react.MessageReactionRemoveEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class HelpRole  extends ListenerAdapter {
+
+    @Override
+    public void onMessageReactionAdd(MessageReactionAddEvent event) {
+
+        if (event.getChannelType() == ChannelType.TEXT) {
```

Comment:
I don't think you need to check if it's a text channel, because I don't think it's possible to even add reactions in a non-text channel.